be/src/vec/data_types/data_type_decimal.cpp

```diff
@@ -230,7 +238,7 @@ const char* DataTypeDecimal<T>::deserialize(const char* buf, MutableColumnPtr* c
         buf = deserialize_const_flag_and_row_num(buf, column, &real_have_saved_num);
 
         // column data
-        auto mem_size = real_have_saved_num * sizeof(T);
+        UInt64 mem_size = real_have_saved_num * sizeof(T);
```

Comment:
if here do cast_set<> maybe we not need change line 251?

Reply:
done